manifests/n/Nonpolynomial/IntifaceCentral/2.6.0/Nonpolynomial.IntifaceCentral.locale.en-US.yaml

```
@@ -0,0 +1,41 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+PackageIdentifier: Nonpolynomial.IntifaceCentral
+PackageVersion: 2.6.0
+PackageLocale: en-US
+Publisher: Nonpolynomial Labs
```

```suggestion
Publisher: Nonpolynomial Labs, LLC
```
This should match Control Panel exactly